### DISCUSSION TITLE
23.03.21 sign in / story 진행상황 (채하은)

### DISCUSSION THREAD

**Discussion by chaehaeun** [withPet-team/withPet-client#94] [Report]
# 🗒️ Description
sign in / story 진행상황

로그인 유지 방식 돌고 돌아 결국 제일 단순한 파베 내장함수 사용하고 끝냈습니다...
시간을 제법 많이 잡아먹어 오늘 열심히 달릴 예정

# ✅ Todo 및 진행 상황
### 진행 상황
- 로그인 유지 방식 변경
- 카드 컴포넌트 마크업 ing
- 본문 더보기 기능 구현
![image](https://user-images.githubusercontent.com/103991310/226772803-ff87e002-f46b-4ed0-85a6-756f38bbce70.png)
![image](https://user-images.githubusercontent.com/103991310/226772833-fc577c26-b7d0-4c6a-9fa7-79b16b80e401.png)

### Todo
- [] 카드 컴포넌트 프롭스 고려해서 마크업 완성하기
- [] 카트 컴포넌트에 다이어리 정보 받아오기

# ETC
화이팅

**Comment by qorbaxk** [comment]
제목 글자수 제한 다이어리에서 만들 때 21자로 제한해 놨어요!

**Reply by chaehaeun** [reply]
오오오 멋져요👍👍👍
